### ISSUE TITLE
Show history of issued codes with status

### ISSUE THREAD

**Issue by whaught** [google/exposure-notifications-verification-server#232]
### TL;DR

Create a page where a user or realm admin can see a list of recently issued codes.

### Design

**Proposal**

- Create a List API for codes.
    - Users get the codes they've issued themselves
    - Realm Admins see all codes
    - Consider pagination and limits

- The original mock calls for showing the
    - test result
    - code
    - expiry
    - anonymized phone (***) ***-**31
        - Note: no. nothing to identify. just uuid.
    - Invalidate code button?

Note: one problem with this (and why we have uuid) is that not all systems will use phone# and may require the user to reach out manually. We don't have any other associated data on who the patient is which makes it tough for the user to know who the code is intended for and follow-up if unclaimed. The uuid allows this or alternative clients to keep such data locally.

**Alternatives considered**

I've created https://github.com/google/exposure-notifications-verification-server/pull/227 where a user can check a single-id at a time. This, however, requires the user to keep a list of their issued UUIDs which may be less useful (the API is possible more useful programmatically rather than as UX) but allows the user to track the private patient data locally and separately. The history table could replace this if its exhaustive.

**Comment by whaught** [comment]
Note: We've since decided to hash the codes. That may make this less useful

**Comment by mikehelmick** [comment]
Would it make sense to show the UUIDs instead?

It would for someone doing an external integration, because they could tie that back into the CRM

**Comment by sethvargo** [comment]
If the use case is external integration, it seems like an API might be more appropriate?

**Comment by mikehelmick** [comment]
both - in general, things we put in the UI should probably also have APIs
